core/db/schema.sql

```diff
@@ -30,20 +37,46 @@ SET default_tablespace = '';
 SET default_with_oids = false;
 
 --
--- Name: messages; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+-- TOC entry 171 (class 1259 OID 24616)
+-- Name: messages; Type: TABLE; Schema: public; Owner: postgres; Tablespace:
 --
 
 CREATE TABLE messages (
     message_id integer NOT NULL,
     title character varying(50) NOT NULL,
     username character varying(50) NOT NULL,
-    posted date DEFAULT ('now'::text)::date,
     message_body character varying(500)
+    posted date DEFAULT ('now'::text)::date
 );
 
 
+ALTER TABLE public.messages OWNER TO postgres;
+
+
+
+-- Table: tasks
+
+-- DROP TABLE tasks;
+
+CREATE TABLE tasks
+(
+  task_id serial NOT NULL,
+  task character varying(80) NOT NULL,
+  username character varying(50) NOT NULL,
+  posted date DEFAULT ('now'::text)::date,
+  --Add a numeric sort
+  CONSTRAINT tasks_pkey PRIMARY KEY (task_id)
+)
+WITH (
+  OIDS=FALSE
+);
+ALTER TABLE tasks
+  OWNER TO postgres;
```

Comment:
Our staging and production servers won't work with the owner set to `postgres` (they have a different account they use). Can you update this schema dump with one that doesn't specify an owner? E.g., `pg_dump --no-owner --schema-only`